### PR TITLE
Hacks for making develcloud6 with SP1 work

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -2521,6 +2524,7 @@ function prepare_proposals()
     iscloudver 6plus && ptfchannel="PTF"
     for machine in $cmachines; do
         ssh $machine "zypper mr -p 90 $ptfchannel"
+        ssh $machine "echo 0 > /proc/sys/kernel/printk"
```
Comment (aplanas) [inline scripts/qa_crowbarsetup.sh]: Maybe copy the same reminder from `qa_crowbarsetup.sh`?

**Comment by aplanas** [comment]
+1

File: scripts/qa_crowbarsetup.sh (hunk 1 above)
Comment (dirkmueller) [inline scripts/qa_crowbarsetup.sh]: sorry, just ignore it, I just pushed my working branch, I'll  rebase on top of the other PR when it got merged

File: scripts/qa_crowbarsetup.sh [hunk 2]
```diff
@@ -1288,9 +1292,8 @@ EOF
         elif iscloudver 6plus; then
             if [ -n "$want_sles12sp1" ] ; then
                 add_ha12sp1_repo
-            else
-                add_ha12_repo
             fi
+            add_ha12_repo
```
Comment (toabctl) [inline scripts/qa_crowbarsetup.sh]: Why do we need the ```12SP0``` HA repos when using ```12SP1``` ?

Comment (dirkmueller) [inline scripts/qa_crowbarsetup.sh]: i had the issue with hacloud and ceph enabled.. since we need to deploy sp0 for SES2, we also needed ha for sp0..

Comment (toabctl) [inline scripts/qa_crowbarsetup.sh]: Does Ceph need packages from HA repos?

Comment (dirkmueller) [inline scripts/qa_crowbarsetup.sh]: no, we just happen to deploy ceph also on ha clusters (4 node setup)

**Comment by toabctl** [comment]
+1

**Pull request merged**